storybook/src/components/Button/Button.stories.tsx

```diff
@@ -12,18 +11,17 @@ const meta = {
   title: 'Components/Buttons/Button',
   component: Button,
   args: {
-    children: 'Button label',
-    variant: 'primary',
+    children: 'Versturen',
     disabled: false,
+    variant: 'primary',
   },
   argTypes: {
```

Comment:
Please add options and labels for `iconPosition`, where the `undefined` option shows as ‘after’, so people can reset it after choosing an option.